packages/web/src/app/doc-pages/tools/team-symbol-doc/team-symbol-generator/team-symbol-generator.component.scss

```diff
@@ -0,0 +1,125 @@
+:host {
+  display: flex;
+  justify-content: center;
+  margin-top: 87px;
+}
+
+.step-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  .image-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 100%;
+    margin-top: 32px;
+    width: 200px;
+    height: 200px;
+    border: 2px dashed var(--e-color-border-2);
+
+    &.color-chosen {
+      border-color: transparent;
+    }
+
+    &.error {
+      border-color: var(--e-color-signal-danger);
+    }
+  }
+}
+
+.generated-colors {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  gap: 24px;
+  padding: 0;
+  margin-top: 24px;
+  margin-bottom: 0;
+
+  .color-circle {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 40px;
+    height: 40px;
+    border-radius: 40px;
+    transition: height 200ms ease-in-out;
```

Comment:
Are these `transition`s copied over from the Color doc by mistake? 😄

Reply:
Yes😅